### PR TITLE
Downloads/index.html Page Added regarding Beautify downloads/index.html #448 issue

### LINKED ISSUE #448: Beautify downloads/index.html
## Overview

When you build this repository locally (or package it for viewing behind a firewall) the page at on-demand/downloads/index.html looks like this:

screen shot 2017-01-23 at 10 22 47 am

To improve the offline experience, we should add some formatting to this page to make it look more like the cheat sheets section at https://services.github.com/resources/#cheat-sheets 

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it! 
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository. 
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

### PR DESCRIPTION
**Pull request by learneradarsh**
Beautify downloads/index.html having sections for cheat sheet download and proper view  and some *.md page renamed into index.md

## Overview
**TL;DR**
Add downloads/index.html for showing cheat sheets in different languages and make it easy to downloads for users.
<https://github.com/github/training-kit/issues/448. Type `closes  #448` to establish a link.>

### Questions
I think you should specify the language names also at the top of web page code.
I've not done it because I'm not sure it's right or wrong according to code standards.
### Next Steps
Create PDF files for rest cheat sheets for making it available for downloads.

### Review
Please give your precious feedback regarding my work.

### CODE REVIEW

File: rvm.sh [hunk 1]
```diff
@@ -0,0 +1,924 @@
+#!/usr/bin/env bash
```
Comment (stoe) [inline rvm.sh]: @learneradarsh, please help me understand, what purpose does the `rvm.sh` file serve in this PR?

I don't think it belongs in this repository.

Comment (learneradarsh) [reply rvm.sh]: Yeah,But other stuff belongs to this repo only

Comment (learneradarsh) [reply rvm.sh]: rvm.sh I need to check it

**Comment by learneradarsh** [comment]
rvm.sh deleted

**Comment by learneradarsh** [comment]
@stoe  Now Can you please review my PR for merge?

**Comment by stoe** [comment]
@learneradarsh, I'll take a look, thanks for the ping.

**Comment by learneradarsh** [comment]
I've changed the file name to index.md because after redirecting to directly it will directly hit to that page only.If you want then we can continue with the previous file name.
Please let me know do you want to go with same or previous file name?

**Comment by ppremk** [comment]
Closing for now since it is stale. Please reopen if needed. Thanks!
